### ISSUE TITLE
Ajouter les tests d'accessibilité systématiques (a11y addon)

### ISSUE THREAD

**Issue by flavius-atticae** [flavius-atticae/shooting-star#193]
## Parent Issue
#183 - Harmonisation des tests et stories Storybook

## Description
Intégrer l'addon a11y de Storybook et s'assurer que chaque story passe les tests d'accessibilité.

## Contexte pour les agents
- Vérifier si @storybook/addon-a11y est installé
- L'ajouter à la config si nécessaire
- Corriger les violations d'accessibilité

## Acceptance Criteria
- [ ] Addon a11y installé et configuré
- [ ] Toutes les stories passent les tests a11y
- [ ] Violations critiques corrigées
- [ ] Documentation sur l'utilisation de l'addon
